src/client/Microsoft.Identity.Client/MigrationAid/MigrationAid.cs

```diff
@@ -808,7 +808,7 @@ public partial class PublicClientApplication
     {
 #if WINDOWS_APP
         /// <summary>
-        /// Flag to enable authentication with the user currently logged-in in Windows.
+        /// Flag to enable authentication with the user currently logged-in on Windows.
```

Comment:
>logged-in [](start = 66, length = 9)

I suppose this is used in many places, wondering if we should call it signed-in everywhere

Reply:
idk

Reply:
done.